static/src/stylesheets/module/external/_from-content-api.scss

```diff
@@ -116,6 +116,11 @@
 .from-content-api pre {
     // Unset from `.from-content-api`
     word-wrap: normal;
+    $x: lighten(colour(guardian-brand-light), 30%);
```

Comment:
I'm not worried about it because the scope is small and the use is clear, but I can give it a better name if you like?